addons/project_issue/models/project_issue.py

```diff
@@ -0,0 +1,335 @@
+        result = project_task_types.sudo(access_rights_uid).name_get()
+        fold = {project_task_type.id: project_task_type.fold for project_task_type in project_task_types}
+        return result, fold
+
+    _group_by_full = {
+        'stage_id': _read_group_stage_ids
+    }
+
+    @api.depends('create_date', 'date_closed', 'date_open')
+    def _compute_day(self):
+        Calendar = self.env['resource.calendar']
+        for issue in self:
+            calendar_id = issue.project_id.resource_calendar_id.id or None
+
+            dt_create_date = fields.Datetime.from_string(issue.create_date)
+            if issue.date_open:
+                dt_date_open = fields.Datetime.from_string(issue.date_open)
+                issue.day_open = (dt_date_open - dt_create_date).total_seconds() / (24.0 * 3600)
+                issue.working_hours_open = Calendar._model.get_working_hours(self.env.cr, self.env.uid, calendar_id, dt_create_date, dt_date_open,
+                                                                             compute_leaves=True, resource_id=False, default_interval=(8, 16),
+                                                                             context=self.env.context)
+
+            if issue.date_closed:
+                dt_date_closed = fields.Datetime.from_string(issue.date_closed)
+                issue.day_close = (dt_date_closed - dt_create_date).total_seconds() / (24.0 * 3600)
+                issue.working_hours_close = Calendar._model.get_working_hours(self.env.cr, self.env.uid, calendar_id, dt_create_date, dt_date_closed,
+                                                                              compute_leaves=True, resource_id=False, default_interval=(8, 16),
+                                                                              context=self.env.context)
+
+    @api.depends('create_date', 'date_action_last', 'date_last_stage_update')
+    def _compute_inactivity_days(self):
+        current_datetime = fields.Datetime.from_string(fields.Datetime.now())
+        for issue in self:
+            dt_create_date = fields.Datetime.from_string(issue.create_date)
+            issue.days_since_creation = (current_datetime - dt_create_date).days
+
+            if issue.date_action_last:
+                issue.inactivity_days = (current_datetime - fields.Datetime.from_string(issue.date_action_last)).days
+            elif issue.date_last_stage_update:
+                issue.inactivity_days = (current_datetime - fields.Datetime.from_string(issue.date_last_stage_update)).days
+            else:
+                issue.inactivity_days = (current_datetime - dt_create_date).days
+
+    @api.onchange('partner_id')
+    def onchange_partner_id(self):
+        """ This function sets partner email address based on partner
+        """
+        self.email_from = self.partner_id.email
+
+    @api.multi
+    def on_change_project(self, project_id):
```

Comment:
on_change_project method does not seem valid. why use multi instead of onchange decorator ?